src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CodeFoldingTests.cs

```diff
@@ -84,67 +84,23 @@ static ImmutableArray<Span> GetMissingExpectedSpans(ICollapsible[] outlines, Imm
             }
         }
 
-        private async Task AssertFoldableBlocksAsync(Span[] foldableSpans)
-        {
-            // TODO: REMOVE THIS METHOD AND UPDATE
-            // ALL TESTS TO USE THE STRING OVERLOAD
-
-            var textView = await TestServices.Editor.GetActiveTextViewAsync(HangMitigatingCancellationToken);
-            var outlines = await GetOutlineRegionsAsync(textView);
-
-            //
-            // Built in retry logic because getting spans can take time.
-            //
-            var tries = 0;
-            const int MaxTries = 10;
-            ImmutableArray<Span> missingSpans;
-            while (tries++ < MaxTries)
-            {
-                missingSpans = GetMissingExpectedSpans(outlines, foldableSpans, textView);
-                if (missingSpans.Length == 0)
-                {
-                    return;
-                }
-
-                Thread.Sleep(100);
-            }
-
-            Assert.Empty(missingSpans);
-
-            static ImmutableArray<Span> GetMissingExpectedSpans(ICollapsible[] outlines, Span[] foldableSpans, ITextView textView)
-            {
-                if (outlines.Length < foldableSpans.Length)
-                {
-                    return foldableSpans.ToImmutableArray();
-                }
-
-                var builder = new List<Span>();
-
-                var spans = outlines.Select(o => o.Extent.GetSpan(textView.TextSnapshot).Span).ToImmutableArray();
-                foreach (var foldableSpan in foldableSpans)
-                {
-                    if (!spans.Contains(foldableSpan))
-                    {
-                        builder.Add(foldableSpan);
-                    }
-                }
-
-                return builder.ToImmutableArray();
-            }
-        }
-
-        [IdeFact(Skip = "Coming in a separate checkin")]
+        [IdeFact]
         public async Task CodeFolding_CodeBlock_Default()
         {
             await TestServices.SolutionExplorer.OpenFileAsync(BlazorProjectName, CounterRazorFile, HangMitigatingCancellationToken);
 
-            await AssertFoldableBlocksAsync(new Span[]
-            {
-                Span.FromBounds(285, 324)
-            });
+            await AssertFoldableBlocksAsync(
+@"@code {
+    private int currentCount = 0;
+
+    private void IncrementCount()
```

Comment:
Shouldn't there be a folding range here too?

Reply:
This is a good point. Let me add a check that the range count matches if all expected are found so we don't miss testing any